conman/routes/tests/test_handlers.py

```diff
@@ -65,6 +66,7 @@ def setUp(self):
         class URLConfRoute(Route):
```

Comment:
Do you think moving the definition into the `setupclass` (or even a `tests.models` module) would make sense?